scripts/scaffold.py

```diff
@@ -229,6 +211,9 @@ def to_snake_case(s):
     s1 = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', s)
     return re.sub('([a-z0-9])([A-Z])', r'\1_\2', s1).lower()
 
+def get_operation_name_in_keys(operation_name, operation_keys):
```

Comment:
support getting operation name that is not pure upper camel case.
for example, from boto3 method `get_hls_stream_url` to operation name `getHLSStreamURL`.